tests/rustdoc-gui/javascript-disabled.goml

```diff
@@ -3,4 +3,9 @@
 javascript: false
 
 go-to: "file://" + |DOC_PATH| + "/test_docs/struct.Foo.html"
+show-text: true
 assert-css: (".sub", {"display": "none"})
+
+// Even though JS is disabled, we should still have themes applied. Links are never black-colored
+// if styles are applied so we check that they are not.
+assert-css-false: ("a.src", {"color": "#000"})
```

Comment:
It might also be possible to add a [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) through [a meta tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#http-equiv) that doesn't allow any script-source.
But I'm not sure if that triggers the noscript alternative.

Reply:
> This could be used to implement a block-network-request: "*.js" command that we could use to simulate a broken network.

Good idea!

> It might also be possible to add a [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) through [a meta tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#http-equiv) that doesn't allow any script-source. But I'm not sure if that triggers the noscript alternative.

It would require to update the DOM before loading the page. Can be done but I'm not a big fan of this approach.